kismet/riscv64.patch

```diff
@@ -0,0 +1,32 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -37,15 +37,18 @@ backup=(
+ install=kismet.install
+ source=(https://www.kismetwireless.net/code/${pkgname}-${_realver}.tar.xz
+         kismet.sysusers
+-        python-install-flags.patch)
++        python-install-flags.patch
++        kismet.patch)
+ sha512sums=('91e86dc1d3b69aa8cb6dff4c4b8bb959dca128f06d1298effe1b23ed24158ce4285abdf2af838f48588b9e71c69f82538d1a002838c4bc1d053868118e394b7a'
+             '452b728c30c9932bdb91741af425267da87a66962ea9b736751948881f5477dee0bff8e2eeca2befdea2920da6d7d9e198bde69cef8c48bb49a0093a7f0ef897'
+-            'c24bff8ecc0288c0584f60760a27b364bf7b013aad809e4050bb6ed703fc60f3ea93f7e897717a6924333ac6734b20b99166846dae4a9c5c303ae1de39f6692c')
++            'c24bff8ecc0288c0584f60760a27b364bf7b013aad809e4050bb6ed703fc60f3ea93f7e897717a6924333ac6734b20b99166846dae4a9c5c303ae1de39f6692c'
++            '054e268f445c70cebcdcbb401c87a9126021daa36eb982907330517f8c015c4289d4056ce12dd0daa6ce114d57390167256a088d92c75f88b599e727332982ba')
+ validpgpkeys=('354689DF3C9DED803381A661D7B28822738BBDB1') # Michael Kershaw
+ 
+ prepare() {
+   cd ${pkgname}-${_realver}
+   patch -Np0 < ../python-install-flags.patch
++  patch -d . -Np0 < ../kismet.patch
+   autoreconf -fiv
+ }
+ 
+@@ -56,7 +59,7 @@ build() {
+     --localstatedir=/var \
+     --sysconfdir=/etc/kismet \
+     --with-suidgroup=315
+-  make all plugins
++  make -j all plugins
```

Comment:
Please do not update this. For devtools compilation, please set MAKEFLAGS in your `$HOME/.makepkg.conf` instead.